modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -102,6 +102,10 @@ def __get_default_jvm_opts(self):
                                                oom_path=os.path.join(self.service.log_dir, "out_of_mem.hprof"),
                                                vm_error_path=os.path.join(self.service.log_dir, "hs_err_pid%p.log"))
 
+        if not self.service.context.globals.get(SAFEPOINT_LOGS_DISABLED, False):
+            default_jvm_opts = merge_jvm_settings(
+                default_jvm_opts, ["-Xlog:safepoint:" + os.path.join(self.service.log_dir, "safepoint.log")])
```

Comment:
Please use more readable settings for log file like:

-Xlog:safepoint*=debug:file=/mnt/service/logs/safepoint.log:time,uptime,level,tags

Reply:
Done